client/src/components/EventPage/EventMeta.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import { Icon, Col } from 'react-materialize';
+import SingleMeta from './SingleMeta';
+import { isEmpty } from '../../utils';
+import { Link } from 'react-router-dom';
+
+
+class EventMeta extends Component {
```

Comment:
What is class EventMeta doing?

Reply:
What if I will say, render Components?
This Component renders meta information for the event.